packages/muelu/research/graham/mf_example_01.cpp

```diff
@@ -528,85 +521,99 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib lib, int arg
       auto rhs_1d = Kokkos::subview (rhs_2d, Kokkos::ALL(), 0);
       SC x = x_left;
       for(size_t i=0; i<n_local; ++i) {
-        rhs_1d(i) = 4*h*h*PI*PI*sin(2*PI*x);
```

Comment:
Could use the math constants from Kokkos..

Reply:
I think when I first started this, some of those constants were in the Kokkos experimental namespace. Are they in a different namespace now?

Reply:
https://kokkos.github.io/kokkos-core-wiki/API/core/numerics/mathematical-constants.html

Reply:
Perfect, thanks! I'll queue that up for next time :)